FirebaseVertexAI/Sources/GenerativeAIService.swift

```diff
@@ -193,10 +193,12 @@ struct GenerativeAIService {
       urlRequest.setValue("Firebase \(authToken)", forHTTPHeaderField: "Authorization")
     }
 
-    // TODO: wait for release approval.
-//    if firebaseInfo.app.isDataCollectionDefaultEnabled {
-//      urlRequest.setValue(firebaseInfo.googleAppID, forHTTPHeaderField: "X-Firebase-AppId")
-//    }
+    if firebaseInfo.app.isDataCollectionDefaultEnabled {
+      urlRequest.setValue(firebaseInfo.googleAppID, forHTTPHeaderField: "X-Firebase-AppId")
+      let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ??
+        "unknown"
```

Comment:
We need to agree on an empty/unknown treatment across all platforms. I would prefer leaving the header unset if the value can't be reliably determined. This way there's only one empty case.

Reply:
SGTM thanks. Updated.